templates/rustdoc/page.html

```diff
@@ -0,0 +1,50 @@
+{%- import "macros.html" as macros -%}
+
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        {# Include any head that rustdoc requires #}
+        {{ rustdoc_head | safe }}
+
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/pure/0.6.0/menus-min.css" type="text/css"
+            media="all" />
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/pure/0.6.0/grids-min.css" type="text/css"
+            media="all" />
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css"
+            type="text/css" media="all" />
+        <link rel="stylesheet" href="/style.css?{{ docsrs_version() | slugify }}" type="text/css" media="all" />
+        <link rel="search" href="/opensearch.xml" type="application/opensearchdescription+xml" title="Docs.rs">
+
+        <title>{{ macros::doc_title(name=krate.name, version=krate.version) }}</title>
+    </head>
+
+    <body>
+        {%- include "rustdoc/navigation.html" -%}
+
+        {# Make the body of the page the documentation generated by rustdoc #}
+        <div id="rustdoc_body_wrapper" class="{{ rustdoc_body_class }}" tabindex="-1">
+            {{ rustdoc_body | safe }}
+        </div>
+
+        <script type="text/javascript" src="/menu.js?{{ docsrs_version() | slugify }}"></script>
+    </body>
+
+    <script>
+        var doc_body = document.getElementById("rustdoc_body_wrapper");
+        if (window.location.hash) {
+            var notFirefox = typeof InstallTrigger === 'undefined';
+            if (notFirefox) {
+                var hash = window.location.hash;
+                window.location.hash = "";
+                setTimeout(function () {
+                    window.location.hash = hash;
+                    doc_body.focus();
+                }, 1);
+            }
+        } else {
+            doc_body.focus();
+        }
+    </script>
+
+</html>
```

Comment:
Missing newline at EOF
```suggestion
</html>
```